lita_config.rb

```diff
@@ -53,7 +53,7 @@
   config.handlers.static_meme.mapping = {
     /^where('| i)s #{config.robot.name}[?]?$/i => "There is no #{config.robot.name}. There is only Zuul.",
     /^thank(s| ?you)\s#{config.robot.name}/i => "At your service.",
-    /top .* the mornin/i => "And the rest of the day to yourself.",
+    /top o.* the mornin/i => "And the rest of the day to yourself.",
```

Comment:
You imply that any of this is serious. Clearly you are mistaken.

`o?` is good enough for me. I'll make the switch.